frontend/www/js/omegaup/arena/location.ts

```diff
@@ -7,6 +7,7 @@ export interface LocationOptions {
   guid: null | string;
   problemAlias: null | string;
   showNewClarificationPopup: boolean;
+  idClarification: null | number;
```

Comment:
nit:

```suggestion
  clarificationId: null | number;
```

por consistencia con todos los demás lugares donde usamos "id" como sufijo.

también, todos estos `clarificationId` sólo están sirviendo para poder enrutar ese número al componente correcto, no? se podría usar el store de clarificaciones en vez de modificar todos los .vue intermedios?

Reply:
Listo, a ver si está bien como le hice.